ui/pages/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -396,6 +402,51 @@ export default class ConfirmTransactionBase extends Component {
       </div>
     ) : null;
 
+    const ledgerInstructionField = isLedgerAccount ? (
+      <div>
+        <div className="confirm-detail-row">
+          <Dialog type="message">
+            <div className="ledger-live-dialog">
+              <Typography
```

Comment:
Wondering if it's worth refactoring this slightly so you don't have to repeat this?

Reply:
I think there is, though I think that would look like defining an array containing the locale keys for the header and steps, and spitting out the `Typography` components inside a `map` run. There would have to be two conditionals inside, one to prepend `-` for only the steps, then checking if the current key is step one and if so, are we not on firefox. I think given this, though this current way is a bit more verbose, it's also a bit more explicit. 

If you had another idea or would prefer to see that, let me know and I can make the change :)

Reply:
Yeah, that's the same route as I was thinking. No worries, just wondering!